tests/test_docs/test_agent_vs_aea/test_agent_vs_aea.py

```diff
@@ -40,7 +40,7 @@ class TestAgentVsAEA(BaseAEATestCase):
     @classmethod
     def setup_class(cls):
         """Setup the test class."""
-        BaseAEATestCase.setup_class()
```

Comment:
Note, we need to be careful not to do this.

Reply:
yes!